src/compute/src/render/join/delta_join.rs

```diff
@@ -411,6 +451,8 @@ where
         inner_as_of.insert(<G::Timestamp>::to_inner(event_time.clone()));
     }
 
+    let could_error = initial_closure.could_error();
```

Comment:
Here we could switch from `unary_fallible` to `unary`. We should find a way that would allow us to not repeat too much of the implementation.